services/vba_documents/UnderstandingDocumentProcessing.md

```diff
@@ -1,7 +1,11 @@
-This document provides some details about how documents are processed by VA and its affiliates after they are submitted via the document upload API. 
+## VA Lighthouse Benefits Intake API - Document Processing
+
+This document provides some details about how documents are processed by VA and its affiliates after they are submitted via the Benefits Intake API.  Please visit the Lighthouse [developer portal](https://developer.va.gov/explore/benefits/docs/benefits) for the latest documentation and release notes.
```

Comment:
One change to add VA before Lighthouse: Please visit the VA Lighthouse [developer portal]

Reply:
done!